typing/typemod.ml

```diff
@@ -2098,6 +2224,34 @@ let report_error ppf = function
       fprintf ppf
         "This is an alias for module %a, which is missing"
         path p
+  | Cannot_hide_id (item_kind, id, kind, id', shadow_loc) ->
+      fprintf ppf
+        "The %s name %s coming from this include cannot be shadowed as it is \
+         used later on by %s %s@ %a"
```

Comment:
I think this message should include break points, to avoid having an error spanning >80 columns as shown in the tests.

Reply:
Oops well sorry, I just noticed it was already being discussed actively!